docs/about.rst

```diff
@@ -3,4 +3,43 @@
 About
 =====
 
-Coming Soon!
+History
+-------
+
+Data scientists spend a huge amount of time on data pre-processing and transformation.
+It would be great (we thought back in the day) to gather the most frequently used data
+pre-processing techniques and transformations in a library, from which we could pick
+and choose the transformation that we need, and use it just like we would use any other
+sklearn class. This was the original vision for Feature-engine.
+
+Feature-engine is an open source Python package originally designed to support the online
+course `Feature Engineering for Machine Learning in Udemy <https://www.udemy.com/feature-engineering-for-machine-learning/?couponCode=FEATENGREPO>`_,
+but has now gained popularity and supports transformations beyond those taught in the
+course. It was launched in 2017, and since then, several releases have appeared and a
+growing international community is beginning to lead the development.
+
+Governance
+----------
+
+The decision making process and governance structure of Feature-engine is laid out in
+the `governance document <https://feature-engine.readthedocs.io/en/latest/governance.html>`_.
+
+Core contributors
+-----------------
+
+The following people are currently core contributors to Feature-engine’s development
+and maintenance:
+
+.. include:: authors.rst
```

Comment:
@ChristopherGS by any chance... do you know how to display 1 github avatar next to the other? as in, one image next to the other in html?

I tried for an hr, and didn't manage. I think I need to add css? can it be?